.github/workflows/ci-core.yml

```diff
@@ -20,72 +20,27 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  golangci-changes:
-    name: detect changes for lint
-    runs-on: ubuntu-latest
-    outputs:
-      src: ${{ steps.golangci-changes.outputs.src }}
-    steps:
-      - name: Checkout the repo
-        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
-      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
-        id: golangci-changes
-        with:
-          filters: |
-            src:
-            - '**/*.go'
-            - '**/go.mod'
-            - '**/go.sum'
-            - '.golangci.yml'
-            - '.github/workflows/ci-core.yml'
-  init:
-    name: initialize
-    runs-on: ubuntu-latest
-    needs: [golangci-changes]
-    defaults:
-      run:
-        shell: bash
-    outputs:
-      # Determine if `on` event should trigger linting to run
-      on_trigger_lint: ${{ steps.golangci-lint.outputs.on_trigger }}
-    steps:
-      - name: Checkout the repo
-        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
-      - name: Check if event should trigger lint
-        id: golangci-lint
-        env:
-          GITHUB_EVENT_NAME: ${{ github.event_name }}
-          GITHUB_BASE_REF: ${{ github.base_ref }}
-          GITHUB_REF: ${{ github.ref }}
-          SRC_CHANGED: ${{ needs.golangci-changes.outputs.src }}
-        run: ./.github/scripts/bash/ontriggerlint.sh | tee -a $GITHUB_OUTPUT
```

Comment:
Want to x this file too?

Reply:
nuked, we can restore the bash testing goodness you created if we need to at any point :)